src/main.ts

```diff
@@ -20,7 +20,7 @@ async function bootstrap() {
     }),
   );
   try {
-    await AppDataSource.initialize();
+    //await AppDataSource.initialize();
```

Comment:
💭 if we've corroborrated that the api works fine without this line maybe we can delete it 👀